index.js

```diff
@@ -26,13 +24,13 @@ module.exports = app => {
   app.on('issue_comment.edited', async context => {
     const regex = /(\-\s\[x\]\s.(.*?)\:)/i
     app.log(context.payload.comment.body)
-    var comment_body = context.payload.comment.body
-    var url = context.payload.issue.html_url
-    var key = ""
+    const comment_body = context.payload.comment.body
+    const url = context.payload.issue.html_url
+    let key = ""
```

Comment:
See this? Before I had no idea which variables would be mutated and which ones would stay the same. Now I can read the code and immediately know! 👍